.trunk/trunk.yaml

```diff
@@ -41,6 +41,8 @@ lint:
       paths:
         - .trunk/configs/cspell.json
         - .gitignore
+        # trunk-ignore(cspell/error)
+        - tests/unit_tests/parsers/conftest.py
```

Comment:
Hang on, can we add the words in the conftest that cpsell is complaining about to the `.trunk/configs/cpsell.json` instead of ignoring the file here please?